temporian/core/operators/simple_moving_average.py

```diff
@@ -43,19 +43,15 @@ def __init__(
 
     self.add_input("data", data)
 
-    features = [  # pylint: disable=g-complex-comprehension
-        Feature(
-            name=f.name(),
-            dtype=f.dtype(),
-            sampling=sampling,
-        )
+    output_features = [  # pylint: disable=g-complex-comprehension
+        Feature(name=f.name(), dtype=f.dtype(), sampling=sampling, creator=self)
```

Comment:
For later: During the e2e execution, with this current code, there should be failure since the SMA operator definition contains a prefix.

For later: What about having a method that automatically generates the prefix using the attribute values.

Reply:
>could you expand on this?

I was referring to the line `Feature(name=f"sma_{f.name()}",` in the SML definition.